apps/src/templates/instructions/InstructionsWithWorkspace.jsx

```diff
@@ -22,6 +22,11 @@ var InstructionsWithWorkspace = React.createClass({
     instructionsMaxHeight: React.PropTypes.number.isRequired,
     setInstructionsHeight: React.PropTypes.func.isRequired,
     setInstructionsMaxHeight: React.PropTypes.func.isRequired,
+
+    isCsf: React.PropTypes.bool,
```

Comment:
Any suggestions? Originally it was just `csf`. I spent some time trying to come up with a better name, and the best I could do was `isCsf` :-P Maybe some terse way of saying "instructions that collapse to a header" vs. "instructions that collapse to a smaller version of themselves"